expression/bench_test.go

```diff
@@ -215,10 +254,14 @@ func fillColumn(eType types.EvalType, chk *chunk.Chunk, colIdx int) {
 			if rand.Float64() < nullRatio {
 				chk.AppendNull(colIdx)
 			} else {
-				if rand.Float64() < 0.5 {
-					chk.AppendInt64(colIdx, -rand.Int63())
+				if len(testCase.geners) > colIdx && testCase.geners[colIdx] != nil {
```

Comment:
Should this be `testCast.geners != nil and testCast.geners[colIdx] != nil`

Reply:
emm, the geners slice can be shorter than the children slice, please the comments about geners.